src/style.css

```diff
@@ -0,0 +1,106 @@
+/*Fonts*/
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: bold;
+}
+
+/*General search styles*/
+
+.form {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background: #fff;
+  box-sizing: border-box;
+  position: relative;
+  margin: 20px 0;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-img {
+  position: absolute;
+  overflow: hidden;
+  z-index: 9;
+}
+
+.form-input {
+  position: relative;
+  border: none;
+  box-sizing: border-box;
+  width: 100%;
+  font-family: Avenir, Helvetica, sans-serif;
```

Comment:
move to `html`

Reply:
Should I use <font> tag?
There is a requirement for this task to use @font-face. But mb I don't understand something. 
Will be gratefull for a hint.

Reply:
you connect styles using @font-face
use html tag for applying fonts to whole document, actually it is the requirement of @alexandra-protyanova
and it is ok to use html tag for this reason